manifests/mod/authz_host.pp

```diff
@@ -0,0 +1,11 @@
+# @summary
+#   Installs `mod_authz_host`.
+# 
+# @see https://httpd.apache.org/docs/current/mod/mod_authz_host.html for additional documentation.
+#
+class apache::mod::authz_host {
+  if versioncmp($apache_version, '2.4') >= 0 {
```

Comment:
Should it have an else clause with `fail()`

Reply:
Like this?